citizenvoice/monkey_patch.py

```diff
@@ -0,0 +1,98 @@
+# monkey_patch.py
+from drf_spectacular.utils import extend_schema
+import types
+
+# Manual mapping of ViewSet classes to their display names
+VIEWSET_MAPPINGS = {
+    'AnswerViewSet': 'Answer',
+    'QuestionViewSet': 'Question', 
+    'SurveyViewSet': 'Survey',
+    'ResponseViewSet': 'Response',
+    'UserViewSet': 'User',
+    'LocationViewSet': 'Location',
+    'PointFeatureViewSet': 'PointFeature',
+    'PolygonFeatureViewSet': 'PolygonFeature',
+    'LineFeatureViewSet': 'LineFeature',
+    'MapViewViewSet': 'MapView',
+    'TopicViewSet': 'Topic',
+}
+
+def patch_viewset_class(cls):
+    """Patch a specific ViewSet class with manual schema decorations."""
+    class_name = cls.__name__
+    
+    # Skip if not in our mapping
+    if class_name not in VIEWSET_MAPPINGS:
+        print(f"Skipping {class_name} - not in mapping")
+        return cls
+    
+    # Skip if already patched
+    if getattr(cls, '_schema_patched', False):
+        print(f"Skipping {class_name} - already patched")
+        return cls
+    
+    model_name = VIEWSET_MAPPINGS[class_name]
+    print(f"Patching {class_name} with model name: {model_name}")
+    
+    # Create new method instances to avoid sharing between ViewSets
+    def create_method_with_schema(original_method, summary, description, operation_id):
+        @extend_schema(
+            summary=summary,
+            description=description,
+            operation_id=operation_id
+        )
+        def new_method(self, request, *args, **kwargs):
+            return original_method(self, request, *args, **kwargs)
+        return new_method
+    
+    # Only patch methods that don't already have custom schema decorations
+    if not hasattr(cls.list, '_spectacular_annotation'):
+        cls.list = create_method_with_schema(
+            cls.list,
+            f"List {model_name}s",
+            f"Retrieve a list of all {model_name}s in the database.",
```

Comment:
Let's avoid to use the words ' lists and database' like in this description. I something like this will be enough: `Retrieves all {model_names}` 
The reason for that is that not all endpoints will return a list (a.k.a. arrays), some do. And 'database' is irrelevant in this context

Reply:
Please also update other patches using those words